src/openqaoa-qiskit/requirements.txt

```diff
@@ -0,0 +1,2 @@
+qiskit>=0.36.1
```

Comment:
Shall we bump this? Or do you think it is safer to keep it at 0.36?

Reply:
As a minimum requirement the code seems to work fine. Though, I personally have never checked if the code works with this version specifically.